test/should_fail/record_index.erl

```diff
@@ -1,6 +1,10 @@
 -module(record_index).
 
--export([f/0, g/0]).
+-export([f/0, g/0,
+         undefined_record/0,
+         undefined_record_infer/0,
```

Comment:
These are not automatically tested as only one fail per module is enough for `should_fail`. We should really fix #129.